src/libraries/System.Text.Json/tests/Common/ExtensionDataTests.cs

```diff
@@ -428,104 +428,93 @@ public override void Write(Utf8JsonWriter writer, CustomOverflowDictionary<JsonE
         }
 
         [Theory]
-        [InlineData(typeof(Dictionary<string, object>), typeof(DictionaryOverflowConverter))]
-        [InlineData(typeof(Dictionary<string, JsonElement>), typeof(JsonElementOverflowConverter))]
-        [InlineData(typeof(CustomOverflowDictionary<object>), typeof(CustomObjectDictionaryOverflowConverter))]
-        [InlineData(typeof(CustomOverflowDictionary<JsonElement>), typeof(CustomJsonElementDictionaryOverflowConverter))]
-        public void ExtensionProperty_SupportsWritingToCustomSerializerWithOptions(Type overflowType, Type converterType)
-        {
-            typeof(ExtensionDataTests)
-                .GetMethod(nameof(ExtensionProperty_SupportsWritingToCustomSerializerWithOptionsInternal), BindingFlags.Static | BindingFlags.NonPublic)
-                .MakeGenericMethod(overflowType, converterType)
-                .Invoke(null, null);
-        }
-
-        private static void ExtensionProperty_SupportsWritingToCustomSerializerWithOptionsInternal<TDictionary, TConverter>()
-            where TDictionary : new()
-            where TConverter : JsonConverter, new()
+        [MemberData(nameof(GetCustomOverflowConverters))]
```

Comment:
Refactored the tests somewhat to reduce the number of generic parameters needed to drive the tests.